glyphs-reader/src/font.rs

```diff
@@ -437,10 +437,10 @@ impl CustomParameters {
     }
 
     fn panose(&self) -> Option<&Vec<i64>> {
-        let Some(CustomParameterValue::Panose(values)) = self.get("panose") else {
-            return None;
-        };
-        Some(values)
+        match self.get("panose").or_else(|| self.get("openTypeOS2Panose")) {
```

Comment:
Add a comment citing https://github.com/googlefonts/glyphsLib/blob/050ef62c52906bc8ca820e7cfc6963138253f41f/Lib/glyphsLib/builder/custom_params.py#L322-L323

For context we've been trying hard to keep the breacrumbs back to python (or C++ or w/e)

Reply:
I've now linked the relevant sections for the names and their precedence

Reply:
I was unable to find this prior to making the PR but have since identified the section of the Glyphs handbook where these longer parameter names are described, in case a later refactor applies them more broadly and needs somewhere to refer back to as well:

https://handbook.glyphsapp.com/custom-parameter-descriptions/
> Custom parameters in camelcase ("exampleName") are defined in the UFO specification and change the font information. In contrast, capitalized ones ("Example Name") are specific to Glyphs and usually change something in the font (for instance, run a filter on the outlines). UFO parameters follow the naming convention for Font Info properties outlined in the UFO 3 specification published in March 2012. Glyphs also makes use of a simplified naming convention. Wherever possible, leave out the prefix of the keyword. For example, instead of "openTypeNameDescription", use "description", or "blueScale" instead of "postscriptBlueScale". Both long and short versions work side by side.

Reply:
Ty!